addons/point_of_sale/static/src/js/printers.js

```diff
@@ -25,16 +25,25 @@ class PrintResultGenerator {
             },
         });
     }
-    IoTResultError() {
+    IoTResultError(printResult) {
+        var body;
+        if (printResult.identifier === undefined)
+        {
+            body = _t('Cannot find a printer.\nYour IoT box may have logged more information concerning this error.');
```

Comment:
`Cannot find a printer` is ambiguous here, an IoT can have several printers. Can't you retrieve with the current context the name of the printer maybe ? If not, maybe prefer a formulation like:
"Cannot find the printer"
or if we know that it is necessarily the receipt printer:
"Cannot find the receipt printer"